proto/storage/cluster.proto

```diff
@@ -343,3 +345,30 @@ message AdmissionControlHealthInfo {
     repeated string status_errors = 3;
 
 }
+
+// LocalScannerHealthInfo carries data about local scanner deployment but does not include local scanner health status
+// derived from this data.
+// Aggregated local scanner health status is not included because it is derived in central and not in the component that
+// first reports LocalScannerHealthInfo (sensor).
+message LocalScannerHealthInfo {
+    // The following fields are made optional/nullable because there can be errors when trying to obtain them and
+    // the default value of 0 might be confusing with the actual value 0. In case an error happens when trying to obtain
+    // a certain field, it will be absent (instead of having the default value).
+
+    oneof total_desired_analyzer_pods_opt {
+        int32  total_desired_analyzer_pods = 1;
+    }
+    oneof total_ready_analyzer_pods_opt {
+        int32  total_ready_analyzer_pods = 2;
+    }
+    oneof total_desired_db_pods_opt {
+        int32 total_desired_db_pods = 3;
+    }
+    oneof total_ready_db_pods_opt {
+        int32  total_ready_db_pods = 4;
+    }
+
+    // Collection of errors that occurred while trying to obtain local scanner health info.
+    repeated string status_errors = 5;
+
```

Comment:
Remove this blank line (and perhaps the one in `AdmissionControlHealthInfo` as well).